lib/daru.rb

```diff
@@ -58,11 +61,21 @@ def create_has_library(library)
         class_variable_get(cv)
       end
     end
+
+    def plotting_library= lib
+      case lib
+      when :gruff, :nyaplot
+        @plotting_library = lib
+      else
+        raise ArgumentError, "Unsupported library #{lib}"
+      end
+    end
```

Comment:
I'm not sure about this whole thing with plotting library selection.

Is there no case when you want Gruff graphs (pretty & static) and Nyaplot graphs (unstable yet dynamic) at the same presentation? And what if we'll add one more graphing library? Always select one and only one? Switch this setting on any action? 

Maybe something like `dataframe.nyaplot.plot(blah)` (and `.gruff.plot(blah)`) would look better?
Or even not try to outsmart ourselves and just use `Daru::Plotting::Gruff.new(dataframe)`?..
The latter approach allows to have other plotting libraries (or even nyaplot and gruff) as separate gems, and any number of `Daru::Plotting::MyCoolPlotter.new(dataframe)` variants.